cpp/src/DataStorm/TopicFactoryI.h

```diff
@@ -15,28 +15,28 @@ namespace DataStormI
 
     class TopicI;
 
-    class TopicFactoryI : public TopicFactory, public std::enable_shared_from_this<TopicFactoryI>
+    class TopicFactoryI final : public TopicFactory, public std::enable_shared_from_this<TopicFactoryI>
```

Comment:
We need a more systematic cleanup in a follow-up PR for final / override and similar.